src/sass/_common/common.scss

```diff
@@ -442,6 +450,15 @@ button .barspinner.white {
     align-self: center;
 }
 
+.child-span {
+    padding: 10px 15px !important;
```

Comment:
any way to avoid this?

Reply:
if we were to use !important, ensure the class is only specific to the usage, hence not in common.scss, can put in account.scss targeting the usage